internal/controllers/addon/upgradepolicy_status.go

```diff
@@ -50,13 +51,32 @@ func (r *AddonReconciler) reportUpgradeStarted(ctx context.Context, addon *addon
 		version  = addon.Spec.Version
 	)
 
-	req := ocm.UpgradePolicyPatchRequest{
+	getReq := ocm.UpgradePolicyGetRequest{
+		ID: policyID,
+	}
+
+	response, err := r.handleGetUpgradePolicy(ctx, getReq)
```

Comment:
```suggestion
	response, err := r.handleGetUpgradePolicyState(ctx, getReq)
```

nit, but I think it's important to distinguish that we are only retrieving the state. Same below.